### DISCUSSION TITLE
インストール版(msi)はどこからダウンロードしたらよいですか？

### DISCUSSION THREAD

**Discussion by hidemiyaAdv** [ffftp/ffftp#397] [Q&A] [answered]
窓の杜からダウンロードすると32bit版も64bit版もポータブル版でした。
インストール版(msi)を配布して頂けませんか？

・窓の杜 - FFFTP
https://forest.watch.impress.co.jp/library/software/ffftp/

**Comment by tt4g** [comment, marked as answer]
https://github.com/ffftp/ffftp/releases の `Assets` からダウンロードできます。

**Reply by hidemiyaAdv** [reply]
ダウンロードできました。ご回答ありがとうございます！
